lesson08/script.js

```diff
@@ -35,9 +34,11 @@ function getList(){
 });
 }
 
-getList().then((value) =>{
+createLoading();
+
+SetDataToLists().then((value) =>{
```

Comment:
The "CreateLoading ();" function shouldn't be included in "SetDataToLists()" function, so I put here.

Reply:
I agree with you!
Considering "PROMISE", I think its position is certainly easy to understand.
I'll fix it again!